c2rust-refactor/scripts/upgrade_ptr_to_ref.lua

```diff
@@ -264,8 +306,6 @@ end
 function Visitor:visit_expr(expr)
```

Comment:
`visit_expr` has gotten pretty long. I'd recommend splitting it up into separate methods for each type of rewrite.